spec/calendar.html

```diff
@@ -711,29 +711,35 @@ <h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-resolveisomonth" aoid="ResolveISOMonth">
-      <h1>ResolveISOMonth ( _fields_ )</h1>
```

Comment:
It would be good to perform this rename separately, for consistency with the other AOs named "ISO...", in the editorial PR, even if we decide not to fix #2497 in this way.

Reply:
I've included this rename in https://github.com/tc39/proposal-temporal/pull/2577, rebasing once that is merged should help reduce the diff in this PR.

Reply:
Rebased.